packages/xo-web/src/common/selectors.js

```diff
@@ -618,3 +618,18 @@ export const getResolvedResourceSets = create(
       }
     })
 )
+
+export const createGetHostState = getHost =>
+  create(
+    (state, props) => getHost(state, props).power_state,
```

Comment:
You can simply get `host` from `props` directly.

Reply:
`createGetHostState` don't know if the host is in the state or in the props